org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Positioner.java

```diff
@@ -98,17 +98,23 @@ public IPosition getPosition() throws ScanningException {
 	}
 
 	@Override
-	public void setMonitors(List<IScannable<?>> monitors) throws ScanningException {
+	public void setMonitors(List<IScannable<?>> monitors) {
 		// TODO Use the _Scannable which is a remote scannable connection.
-		throw new ScanningException("Monitors may not be set on a remote positioner!");
+		throw new UnsupportedOperationException("Monitors may not be set on a remote positioner!");
 	}
 
 	@Override
-	public void setMonitors(IScannable<?>... monitors) throws ScanningException {
+	public void setMonitors(IScannable<?>... monitors) {
 		setMonitors(Arrays.asList(monitors));
 	}
 
 	@Override
+	public void setScannables(List<IScannable<?>> scannables) {
+		// TODO Use the _Scannable which is a remote scannable connection.

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)